.buildkite/release-pipeline.yaml

```diff
@@ -1,4 +1,22 @@
 steps:
+  # aarch64 + CUDA builds
+  # NOTE: PyTorch only has aarch64 builds for CUDA 12.9
```

Comment:
Is this for torch 2.7.0? Does this change for 2.7.1 or 2.8.0?

Reply:
This is still the case for all PyTorch releases today afaik